mw-webapp/src/component/icon/Icon.tsx

```diff
@@ -60,5 +60,9 @@ export const Icon = (props: IconProps) => {
     props.className,
   );
 
-  return IconDictionary[props.name]({...props, className});
+  return (
+    <a>
+      {IconDictionary[props.name]({...props, className})}
+    </a>
+  );
```

Comment:
Can you please explain why Icon is a link?

Reply:
Icon is not a link, it should be wrapped in a button

Reply:
This is my mistake. It would have been more correct to use the <button> tag and wrap it with the Icon component. I'll fix it now.

Reply:
> Icon is not a link, it should be wrapped in a button

Icon is not always button

Reply:
ok, i understand, i need to remove wrapper in Icon component, and create wrapper button over Icon. Fixed it in the last commit